src/api/data/init.ts

```diff
@@ -12,6 +12,22 @@ export function initDatabase(redo: boolean = false): void {
   // Connect to the SQLite database
   const db = new Database("app.db");
 
+  // create settings table
+  const createSettingsTable = db.prepare(`
```

Comment:
<img width="912" alt="image" src="https://github.com/dongmingli-Ben/email-wizard-desktop/assets/65433575/87987e9b-369c-43fe-bc9c-0ad6dc0bca2d">

creating settings table seems failed in my enviornment

Reply:
solved, just delete and reboot the app